pkg/destroy/azure/azure.go

```diff
@@ -331,7 +378,7 @@ func toRecordType(t string) dns.RecordType {
 
 func deleteResourceGroup(ctx context.Context, client resources.GroupsClient, logger logrus.FieldLogger, name string) error {
 	logger = logger.WithField("resource group", name)
-	ctx, cancel := context.WithTimeout(ctx, 30*time.Minute)
+	ctx, cancel := context.WithTimeout(ctx, 60*time.Minute)
```

Comment:
30 minutes is more than enough for one try. Since outer context is longer we can retry...